connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/CheckConnectJvmClientCompatibility.scala

```diff
@@ -126,7 +126,11 @@ object CheckConnectJvmClientCompatibility {
       IncludeByName("org.apache.spark.sql.RuntimeConfig.*"),
       IncludeByName("org.apache.spark.sql.TypedColumn.*"),
       IncludeByName("org.apache.spark.sql.SQLImplicits.*"),
-      IncludeByName("org.apache.spark.sql.DatasetHolder.*"))
+      IncludeByName("org.apache.spark.sql.DatasetHolder.*"),
+      IncludeByName("org.apache.spark.sql.streaming.DataStreamReader.*"),
```

Comment:
also need include `org.apache.spark.sql.streaming.StreamingQueryProgress`

Reply:
`StreamingQueryProgress` is not complete yet. 
We are just saving the JSON.  We are excluding all the methods below with `TODO(SPARK-43128)`